### DISCUSSION TITLE
Busy

### DISCUSSION THREAD

**Discussion by Foxes321** [The-Wineskin-Project/WineskinServer#548] [Q&A]
When I try to install a .exe file into a wrapper, it goes to "Wineskin is currently busy please wait". I have waited almost an hour for it to work. I've tried making new wrappers, trying different versions, restarting my computer. please help.

**Comment by Gcenx** [comment]
You’ve really not provided any useful information, you need to provide some basics at least.

- macOS version?
- What Engine was being used?
- What are you trying to install?

**Reply by Foxes321** [reply]
sorry, I'm rather new to this.
-macos verison 12.7.3
-Engine WS11WineCX64Bit23.7.1
-Attempting to download Undertale Yellow.exe

**Reply by Foxes321** [reply]
please help?

**Reply by Gcenx** [reply]
Open the installer inside the wrapper drive_c directory and try running it again, if it still doesn’t work I’d need LastRunWine.log

**Reply by Foxes321** [reply]
im so sorry but i have no idea how to do that.

**Reply by Foxes321** [reply]
[LastRunWine.log](https://github.com/Gcenx/WineskinServer/files/14473927/LastRunWine.log)

**Reply by Gcenx** [reply]
That log only contains a single line

> wineserver: Can't check in server_mach_port

Ether you have other wine based things running or something else is simply blocking wine from functioning.

**Reply by Foxes321** [reply]
I have no other wine based things running, if you have any idea what's blocking it I would love to know.

**Reply by Foxes321** [reply]
update: tried doing a test run and got this message:
![Screen Shot 2024-03-16 at 3 10 59 PM](https://github.com/Gcenx/WineskinServer/assets/161523122/aa229466-5c19-48e3-b211-9c7fb8308985)

**Reply by Foxes321** [reply]
do you know whats blocking it?

**Comment by emendelson** [comment]
@Foxes321 - Ctrl-click on the wrapper. 

Choose Show Package Contets. 

In the search field at the upper right of the Finder window, type  `LastRunWine` then select from the dropdown `Name contains`, and, in the two tabs, select the wrapper, not "This Mac."

You'll find the file.  Drag it into the field where you're typing an answer.

**Reply by Foxes321** [reply]
thank you!
